README.md

```diff
@@ -1,3 +1,17 @@
+# Apigee Remote Policy Execution 
+The purpose of this project is to provide a proof of concept and protocol for
+running Apigee policies on remote HTTP servers. The `execute.proto` file in
+`callout/proto` defines a protocol buffer schema for encoding the information
```

Comment:
the proto technically is not only for java callouts.  Is it possible to rename the directory to something like 'execution'?

Reply:
oh, that was my mistake in the README. the proto file is actually in the `proto` folder which is outside of `callout`